raiden/transfer/node.py

```diff
@@ -447,17 +444,16 @@ def handle_delivered(node_state, state_change):
 
 
 def handle_new_token_network(node_state, state_change):
-    events = list()
-
     token_network_state = state_change.token_network
     payment_network_identifier = state_change.payment_network_identifier
-    payment_network = node_state.identifiers_to_paymentnetworks.get(payment_network_identifier)
 
-    if payment_network is not None:
-        tokens_to_networks = payment_network.tokenidentifiers_to_tokennetworks
-        tokens_to_networks[token_network_state.address] = token_network_state
+    maybe_add_tokennetwork(
+        node_state,
+        payment_network_identifier,
+        token_network_state,
+    )
 
-    # TODO: add ContractSend
```

Comment:
Remember what this TODO was referring to? I guess this state iteration is never supposed to produce any events.

Reply:
It doesn't make sense to have a `ContractSend` here, it's more likely a bad comment I left behind while adding the `node.py` module.